src/pineko/check.py

```diff
@@ -106,58 +140,25 @@ def is_fonll_b(fns, lumi):
     return False
 
 
-def contains_fact(grid, max_as, max_al):
-    """Check whether fact scale-variations are available.
-
-    The order is specified by max_as and max_al.
-
-    Parameters
-    ----------
-    grid : pineappl.grid.Grid
-           Pineappl grid
-    max_as : int
-             max as order
-    max_al : int
-             max al order
-    Returns
-    -------
-    bool
-        is fact scale-variation available for as
-    bool
-        is fact scale-variation available for al
-    """
+def get_order_list(grid, max_as, max_al):
+    """Return the order list after filtering according to max_as and max_al."""
```

Comment:
Just because of my aversion for redundant names (let's save characters, I'll take less to review!) I'd really drop the `get_` in the name, and even the `Return` in the docstring.

```suggestion
def orders(grid, max_as, max_al):
    """Select the relevant orders. 
    
    The orders in the grid are filtered according to `max_as` and `max_al`.
    
    """
```

Moreover, `_list` is just explicating the return type. If that is relevant, I would suggest to use type hints:
```py
def orders(grid, max_as, max_al) -> list:
```

Reply:
[2857085](https://github.com/NNPDF/pineko/pull/75/commits/2857085498191b8b5a650a2333b81f184ce89271)